queries/python.scm

```diff
@@ -652,3 +652,8 @@ operator: [
 (function_definition
   "->" @disqualifyDelimiter
 )
+
+(
+  (string_start) @pairDelimiter
+  (#match? @pairDelimiter "^\\w")
```

Comment:
No need to qualify the default delimiters

Reply:
```suggestion
  (#match? @pairDelimiter "^[a-zA-z]+")
```

We don't need numbers since the Python special string prefixes can only be letters